examples/hosts/app-integration/container-views/webpack.config.js

```diff
@@ -34,6 +35,9 @@ module.exports = env => {
             libraryTarget: "umd"
         },
         plugins: [
+            new webpack.ProvidePlugin({
+                process: 'process/browser'
+            }),
```

Comment:
Is this related to switching to the model loading system? Or was this just an unrelated update?

Reply:
Unrelated - I was doing basic before/after manual testing with my changes and realized it was completely broken.  Same goes for the propertyDDS test that wasn't rendering.